client/jetpack-cloud/sections/partner-portal/primary/invoices-dashboard/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslate } from 'i18n-calypso';
+import { ReactElement } from 'react';
+import CardHeading from 'calypso/components/card-heading';
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import InvoicesList from 'calypso/jetpack-cloud/sections/partner-portal/invoices-list';
+import SelectPartnerKeyDropdown from 'calypso/jetpack-cloud/sections/partner-portal/select-partner-key-dropdown';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+import './style.scss';
+
+export default function InvoicesDashboard(): ReactElement {
+	const translate = useTranslate();
+
+	return (
+		<Main wideLayout className="invoices-dashboard">
+			<DocumentHead title={ translate( 'Invoices' ) } />
+			<SidebarNavigation />
+
+			<div className="invoices-dashboard__header">
+				<CardHeading size={ 36 }>{ translate( 'Invoices' ) }</CardHeading>
+
+				<SelectPartnerKeyDropdown />
```

Comment:
**Question**

Would it make sense to have a `All keys` option?
IMO it probably wouldn't since agencies only have one key (their production key), but if this page will become accessible to other types of partners as well, then some of these have multiple keys for different contexts.
I don't believe it's worth the potential effort - especially since we're focusing on agencies right now - because the whole "select key" experience would be different here, and what key should be used when they select `Licenses` in the menu? 😅

Reply:
The select key thing was shipped as a stop-gap thing in record time just so we could get things moving - it was not looked into by Design since we don't have a valid use case right now apart from a12s.